mgmtsystem_action/views/mgmtsystem_action.xml

```diff
@@ -0,0 +1,194 @@
+<odoo>
+    <data>
+
+        <record id="view_mgmtsystem_action_tree" model="ir.ui.view">
+            <field name="name">mgmtsystem.action.tree</field>
+            <field name="model">mgmtsystem.action</field>
+            <field name="type">tree</field>
+            <field name="arch" type="xml">
+                <tree string="Action">
+                    <field name="reference"/>
+                    <field name="name" string="Subject"/>
+                    <field name="type_action"/>
+                    <field name="user_id"/>
+                    <field name="date_deadline"/>
+                    <field name="stage_id"/>
+                    <field name="company_id" groups="base.group_multi_company"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="view_mgmtsystem_action_filter" model="ir.ui.view">
+            <field name="name">mgmtsystem.action.filter</field>
+            <field name="model">mgmtsystem.action</field>
+            <field name="type">search</field>
+            <field name="arch" type="xml">
+                <search string="Action">
+                    <group>
+                        <separator orientation="vertical"/>
+                        <field name="reference"/>
+                        <field name="date" select='1'/>
+                        <field name="user_id"/>
+                        <field name="system_id"/>
+                        <field name="company_id" groups="base.group_multi_company"/>
+                    </group>
+                    <newline/>
+                    <group expand="0" string="Group By...">
+                        <filter string="User" icon="terp-personal" domain="[]" context="{'group_by':'user_id'}"/>
+                        <separator orientation="vertical"/>
+                        <filter string="Type" icon="terp-personal" domain="[]" context="{'group_by':'type_action'}"/>
+                    </group>
+                </search>
+            </field>
+        </record>
+
+        <record id="view_mgmtsystem_action_form" model="ir.ui.view">
+            <field name="name">mgmtsystem.action.form</field>
+            <field name="model">mgmtsystem.action</field>
+            <field name="arch" type="xml">
+                <form string="Action">
+                    <header>
+                        <field name="stage_id" widget="statusbar" clickable="True"/>
+                    </header>
+                    <sheet string="Action">
+                        <group colspan="4" col="4">
+                            <field name="name" colspan="4" string="Subject"/>
+                            <field name="type_action" required="True"/>
+                            <field name="reference"/>
+                            <field name="user_id" string="Responsible" required="1"/>
```

Comment:
IMO we should set the fields as required directly in the Model Field, instead of here.